RockPaperScissors/RockPaperScissors/main.swift

```diff
@@ -6,5 +6,47 @@
 
 import Foundation
 
-print("Hello, World!")
+func inputMenu() {
+    print("가위(1), 바위(2), 보(3)! <종료 : 0>: ", terminator: "")
+    guard let inputNumber = readLine() else { return }
+    guard let userCard = Int(inputNumber) else { return }
```

Comment:
guard let을 이용해 옵셔널 바인딩을 잘 해주셨네요 👍

참고로 각각의 guard else 구문을 통해 예외 처리 타이밍을 나누고 싶지 않으시다면 조건을 `,`로 연결할 수 있습니다 🙂
```suggestion
    guard let inputNumber = readLine(),
          let userCard = Int(inputNumber) else { return }
```

Reply:
guard문 ,이용해서 수정하였습니다.